homeassistant/components/screenlogic/binary_sensor.py

```diff
@@ -3,13 +3,18 @@
 
 from screenlogicpy.const import ON_OFF
 
-from homeassistant.components.binary_sensor import DEVICE_CLASSES, BinarySensorEntity
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_PROBLEM,
+    BinarySensorEntity,
+)
 
 from . import ScreenlogicEntity
 from .const import DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
+SL_DEVICE_TYPE_TO_HA_DEVICE_CLASS = {"alarm": DEVICE_CLASS_PROBLEM}
```

Comment:
It would be a bit easier to maintain long term if `"alarm"` was a constant imported from `screenlogicpy` which would allow changes to the value without breaking anything

Reply:
Same with the other sensor platform

Reply:
Commit https://github.com/home-assistant/core/pull/48136/commits/2d488e2a1747d24006a99a3b5f374c69f73748c7